task.go

```diff
@@ -282,8 +282,9 @@ func (e *Executor) runCommand(ctx context.Context, t *taskfile.Task, call taskfi
 			return fmt.Errorf("task: failed to get variables: %w", err)
 		}
 		stdOut, stdErr, close := outputWrapper.WrapWriter(e.Stdout, e.Stderr, t.Prefix, outputTemplater)
+		var errored bool = false
 		defer func() {
-			if err := close(); err != nil {
+			if err := close(errored); err != nil {
```

Comment:
To avoid having to have an outer variable, I believe it's probably fine to move this block out of the `defer` block and after `execext.RunCommand`.

Reply:
defer block moved after command execution, done.